docs/pages/code/operations/max.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App;
+
+use loophp\collection\Collection;
+use stdClass;
+
+include __DIR__ . '/../../../../vendor/autoload.php';
+
+$callback = static fn (stdClass $left, stdClass $right): stdClass => max($left->age, $right->age);
```

Comment:
This callback is invalid, it returns a `int` instead of a `stdClass`, therefore, the code example is erroneous as well.